connectors/SignRequest/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "capabilities": [
+      "actions",
+      "triggers"
+    ],
+    "connectionParameters": {
+      "token": {
+        "oAuthSettings": {
+          "clientId": "Cg13CyJLi6ibicOYMweTl12uI1X6Rq6FTb7j9Y3v",
```

Comment:
Can you please change this to a dummy client id? For example, a string such as this one is more suitable {{CLIENT_ID}}